tests/test_file.py

```diff
@@ -45,6 +45,18 @@ def test_locale(self, mozwebqa):
             print "There is no language selector on the page"
 
     @pytest.mark.nondestructive
+    def test_locale_zh_TW (self, mozwebqa):
+        main_page = MySiteHomePage(mozwebqa)
```

Comment:
We should include a docstring to explain the test here. If I understand correctly we want to check that opening [base]/zh_TW should either stay as [base]/zh_TW or become [base]/en-US. If this is the case I would attempt to make this test a little more generic using a list of locales that this could apply to (with just one for now). I would therefore also update the name of the test to reflect this. Something like `test_locales_persist_or_fallback_to_en_us`.